client/my-sites/stats/videopress-stats-module/index.jsx

```diff
@@ -0,0 +1,260 @@
+import { Component } from 'react';
+import { connect } from 'react-redux';
+import SectionHeader from 'calypso/components/section-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import {
+	isRequestingSiteStatsForQuery,
+	getVideoPressPlaysComplete,
+} from 'calypso/state/stats/lists/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import DatePicker from '../stats-date-picker';
+import DownloadCsv from '../stats-download-csv';
+import ErrorPanel from '../stats-error';
+import StatsModulePlaceholder from '../stats-module/placeholder';
+
+import '../stats-module/style.scss';
+import './style.scss';
+
+class VideoPressStatsModule extends Component {
+	static propTypes = {
+		summary: PropTypes.bool,
+		moduleStrings: PropTypes.object,
+		period: PropTypes.object,
+		path: PropTypes.string,
+		siteSlug: PropTypes.string,
+		siteId: PropTypes.number,
+		data: PropTypes.object,
+		query: PropTypes.object,
+		statType: PropTypes.string,
+		showSummaryLink: PropTypes.bool,
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		showSummaryLink: false,
+		query: {},
+	};
+
+	state = {
+		loaded: false,
+	};
+
+	componentDidUpdate( prevProps ) {
+		if ( ! this.props.requesting && prevProps.requesting ) {
+			// eslint-disable-next-line react/no-did-update-set-state
+			this.setState( { loaded: true } );
+		}
+
+		if ( this.props.query !== prevProps.query && this.state.loaded ) {
+			// eslint-disable-next-line react/no-did-update-set-state
+			this.setState( { loaded: false } );
+		}
+	}
+
+	getModuleLabel() {
+		if ( ! this.props.summary ) {
+			return this.props.moduleStrings.title;
+		}
+		const { period, startOf } = this.props.period;
+		const { path, query } = this.props;
+
+		return (
+			<DatePicker
+				period={ period }
+				date={ startOf }
+				path={ path }
+				query={ query }
+				summary={ true }
+			/>
+		);
+	}
+
+	getHref() {
+		const { summary, period, path, siteSlug } = this.props;
+
+		// Some modules do not have view all abilities
+		if ( ! summary && period && path && siteSlug ) {
+			return (
+				'/stats/' +
+				period.period +
+				'/' +
+				path +
+				'/' +
+				siteSlug +
+				'?startDate=' +
+				period.startOf.format( 'YYYY-MM-DD' )
+			);
+		}
+	}
+
+	render() {
+		const {
+			className,
+			summary,
+			path,
+			data,
+			moduleStrings,
+			requesting,
+			statType,
+			query,
+			period,
+			siteSlug,
+		} = this.props;
+
+		let completeVideoStats = [];
+		if ( data && data.days ) {
+			completeVideoStats = Object.values( data.days )
+				.map( ( o ) => o.data )
+				.flat();
+		}
+
+		const noData = data && this.state.loaded && ! completeVideoStats.length;
+		// Only show loading indicators when nothing is in state tree, and request in-flight
+		const isLoading = ! this.state.loaded && ! ( data && data.length );
+
+		// TODO: Support error state in redux store
```

Comment:
Errant `TODO` - not sure we planned to do this or it was copy-pastad?

Reply:
Probably 🍝 ?

Reply:
Verified that it was copied, removed in https://github.com/Automattic/wp-calypso/pull/63173/commits/5ef4172f29c6323a4c9e91647f9b48bac07a02dc